pkg/function/utils.go

```diff
@@ -100,7 +100,7 @@ func ValidateProfile(profile *param.Profile) error {
 }
 
 // GetPodWriter creates a file with Google credentials if the given profile points to a GCS location
-func GetPodWriter(cli kubernetes.Interface, ctx context.Context, namespace, podName, containerName string, profile *param.Profile) (*kube.PodWriter, error) {
+func GetPodWriter(cli kubernetes.Interface, ctx context.Context, namespace, podName, containerName string, profile *param.Profile) (kube.PodWriter, error) {
```

Comment:
Do you think, we can move the context to the first position in the argument list? Context should usually come first according to the convention.